corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -203,4 +206,45 @@ public int dropTable(String namespace, String tablename) {
         log.info("\n======================\n");
         return tableSize;
     }
+
+    /**
+     * Loads the table with random data
+     * @param namespace - the namespace where the table belongs
+     * @param tablename - table name without the namespace
+     * @param numItems - total number of items to load
+     * @param batchSize - number of items in each transaction
+     * @return - number of entries in the table
+     */
+    public int loadTable(String namespace, String tablename, int numItems, int batchSize) {
+        verifyNamespaceAndTablename(namespace, tablename);
+        CorfuStore store = new CorfuStore(runtime);
+        try {
+            TableOptions.TableOptionsBuilder<Object, Object> optionsBuilder = TableOptions.builder();
+            if (diskPath != null) {
+                optionsBuilder.persistentDataPath(Paths.get(diskPath));
+            }
+            store.openTable(namespace, tablename,
+                    TableName.getDefaultInstance().getClass(),
+                    TableName.getDefaultInstance().getClass(),
+                    TableName.getDefaultInstance().getClass(),
+                    optionsBuilder.build());
+
+            TableName dummyVal = TableName.newBuilder().setNamespace(namespace).setTableName(tablename).build();
+            log.info("Loading {} items in {} batchSized transactions into {}${}",
+                    numItems, batchSize, namespace, tablename);
+            while (numItems > 0) {
+                log.info("loadTable: Items left {}", numItems);
+                TxBuilder tx = store.tx(namespace);
+                for (int j = batchSize; j > 0 && numItems > 0; j--, numItems--) {
+                    TableName dummyKey = TableName.newBuilder().setNamespace(Integer.toString(numItems))
+                            .setTableName(Integer.toString(j)).build();
+                    tx.update(tablename, dummyKey, dummyVal, dummyVal);
+                }
+                tx.commit();
+            }
+        } catch (Exception e) {
+            log.error("loadTable: {} {} {} {} failed.", namespace, tablename, numItems, batchSize, e);
+        }
+        return numItems/batchSize;
```

Comment:
Probably it should be (int)Math.ceil((double)numItems/batchSize);
Imagine 10 entries, in batches of 8, it would return 1, instead of 2.

Reply:
done. thanks